### PR TITLE
Add encryption flag to config, template, and adapter.

### PR DESCRIPTION
**Pull request by aeberlin**
Let me know if I'm missing something.

### CODE REVIEW

**Comment by aeberlin** [comment]
@chrismccord Can we please get this merged? It's a really simple addition.

**Comment by chrismccord** [comment]
<3 <3 <3

File: app/assets/javascripts/sync.coffee.erb [hunk 1]
```diff
@@ -7,6 +7,7 @@ $ = jQuery
   FAYE_HOST: "<%= Sync.server %>"
   PUSHER_API_KEY: "<%= Sync.api_key %>"
   CLIENT_ADAPTER: "<%= Sync.adapter %>"
+  ENCRYPTION_FLAG: "<%= Sync.encryption_flag %>"
```
Comment (ajb) [inline app/assets/javascripts/sync.coffee.erb]: This is a boolean, right? Does it need to be in quotes?

**Comment by aeberlin** [comment]
TY!

File: app/assets/javascripts/sync.coffee.erb (hunk 1 above)
Comment (aeberlin) [inline app/assets/javascripts/sync.coffee.erb]: The adapter wasn't taking a boolean on my end. I'll send a follow-up if that's no longer the case.

**Pull request merged**